### PR TITLE
[add] - navigation bar

### PR DESCRIPTION
**Pull request by gideonnnalue**
**Desc:**

- Added navigation bar
- Added navigation brand logo
- Added navigation items

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -13,6 +14,8 @@ function App() {
                 <Switch>
                     <Route path="/signup" component={Signup} />
                     <Route path="/signin" component={Signin} />
+                    <Route path="/feeds" component={Feed} />
+                    <Route path="/" component={Feed} />
                 </Switch>
             </Router>
```
Comment (EngrEric) [inline src/App.jsx]: it is better to separate this to a component

Comment (gideonnnalue) [reply src/App.jsx]: how do u mean boss

Comment (EngrEric) [reply src/App.jsx]: create a new component called `Route` and import it here

Comment (SapeleD3) [reply src/App.jsx]: I think it  is fairly understandablee and still static

**Pull request merged**